docs/ARM.md

```diff
@@ -65,7 +65,7 @@ knife azurerm server create
   --azure-image-os-type centos
   --azure-image-reference-sku '6.5'
   --azure-vm-size Standard_A1_v2
-  --ssh-user myuser --ssh-password mypassword
+  --connection-user myuser --connection-password mypassword
   -r "recipe[cbk1::rec1]"
   -c ~/.chef/knife.rb
```

Comment:
we should update the docs to refer to config.rb as the config.rb file is a replacement for the knife.rb file, starting with the Chef Client 12.0 release.